src/install/haproxy/windows/install-msi.mdx

```diff
@@ -0,0 +1,10 @@
+---
+componentType: default
+headingText: Download using MSI
+---
+
+1. Download the latest .MSI installer image for the desired integration [from our repository](https://download.newrelic.com/infrastructure_agent/windows/integrations/).
+2. In an admin account, run the install script using an absolute path.
```

Comment:
Add space. Also, you indent here for this codeblock and i doooo think it looks a little nicer so might consider indenting all the code blocks